src/js/common/utils/helpers.js

```diff
@@ -101,14 +101,33 @@ export function dateToMoment(dateField) {
   });
 }
 
+export function dropTimeAndTimezone(date) {
+  let newDate = moment(date).format();
+  const indexOfTime = newDate.indexOf('T');
```

Comment:
Whoops, you're right, my mistake.  Ironically, this code is still problematic.  I just ran a test case on this function:
```javascript
> dropTimeAndTimezone('1995-11-12T03:00:00.000+0000');
"1995-11-11"
// expected = 1995-11-12
```